streamz/sources.py

```diff
@@ -228,15 +233,18 @@ class from_http_server(Source):
     start : bool
         Whether to immediately startup the server. Usually you want to connect
         downstream nodes first, and then call ``.start()``.
+    server_kwargs : dict or None
+        If given, set of further parameters to pass on to HTTPServer
 
     Example
     -------
     >>> source = Source.from_http_server(4567)  # doctest: +SKIP
     """
 
-    def __init__(self, port, path='/.*', start=False):
+    def __init__(self, port, path='/.*', start=False, server_kwargs=None):
         self.port = port
         self.path = path
+        self.server_kwargs = server_kwargs or {}
```

Comment:
In one case you use `tcp_kwargs` and in the other `server_kwargs`.  Maybe use `server_kwargs` in both places for consistency of the API?